zenodo/modules/pages/utils.py

```diff
@@ -81,9 +110,47 @@ def send_support_email(context):
         body=msg_body
     )
 
-    if context['form'].attachments.data:
-        msg.attach(context['form'].attachments.data.filename,
-                   'application/octet-stream',
-                   context['form'].attachments.data.read())
+    attachments = request.files.getlist("attachments")
+    if attachments:
+        if check_attachment_size(attachments):
+            for upload in attachments:
+                msg.attach(upload.filename,
+                           'application/octet-stream',
+                           upload.read())
+        else:
+            return "error"
 
     mail.send(msg)
+
+
+def os_browser(info):
+    """Function to return string."""
+    info_version = '.'.join(
+        [v for v in (info['major'], info['minor'], info['patch']) if v]
```

Comment:
All these fields might be missing from the dict. You shold use `info.get('major')` instead of direct index access, especially since this information comes from a third-party library